### PR TITLE
added smoth scroll

### PR DESCRIPTION
**Pull request by HKrzyszczak**

### CODE REVIEW

File: js/animations.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+$(document).ready(function () {
+    $('a.nav__link').click(function() {
+
+            event.preventDefault();
+            $('body').animate({
```
Comment (marcindomanski1) [inline js/animations.js]: The smooth scroll works for me only after that line is modified to:
`$('html, body').animate({`

File: index.html [hunk 2]
```diff
@@ -211,17 +211,17 @@ <h4 class="heading">Stay connected</h4>
...
             </div>
             <div class="row ft__bottom">
                 <div class="col-xs-12 col-sm-8 col-md-6 col-lg-6">
                     <ul class="footer_nav">
-                        <li><a href="#sct1">About app</a></li>
-                        <li><a href="#sct2">Main features</a></li>
-                        <li><a href="#sct3">Stay connected</a></li>
+                        <li><a class="nav__link" href="#sct1">About app</a></li>
+                        <li><a class="nav__link" href="#sct2">Main features</a></li>
+                        <li><a class="nav__link" href="#sct3">Stay connected</a></li>
```
Comment (marcindomanski1) [inline index.html]: Class "nav__link" changed the 'display' value from 'inline' to 'block'.

File: index.html [hunk 3]
```diff
@@ -211,17 +211,17 @@ <h4 class="heading">Stay connected</h4>
             <div class="row ft__top">
                 <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
                     <h4>Application developed by:</h4>
-                    <a href="http://www.redcoders.pl" target="_blank">
+                    <a href="http://www.redcoders.pl" target="_blank" class="nav__link">
```
Comment (marcindomanski1) [inline index.html]: It stopped redirecting to redcoders.pl

File: index.html [hunk 4]
```diff
@@ -59,7 +59,7 @@
                     </div>
                     <div class="collapse navbar-collapse" id="yk-navbar-collapse">
                         <ul class="nav navbar-nav">
-                            <li><a class="btn nav__link btn--rounded" href="#sct1">About app</a></li>
+                            <li><a class="btn nav__link btn--rounded internal" href="#sct1">About app</a></li>
```
Comment (marcindomanski1) [inline index.html]: I do not see any use for that new class at this point.

**Review by marcindomanski1 — CHANGES_REQUESTED** [review]
4 issues need to be addressed. The details are included in my comments.

**Comment by HKrzyszczak** [comment]
Now should be ok. Please review once again.

**Review by marcindomanski1 — APPROVED** [review]

**Pull request merged**